scala-setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -eu
+
+SCALA_CLI_BASE_URL="https://github.com/VirtuslabRnD/scala-cli/releases/download/latest/"
+
+architecture() {
+    
+    UNAME="$(uname)"
+    
+    case "$UNAME" in
+        Linux)
+            OS_NAME=pc-linux
+            ;;
+        Darwin)
+            OS_NAME=apple-darwin
+            ;;
+        *)
+            err "unrecognized OS type: $UNAME"
```

Comment:
It seems `err` isn't defined.

Reply:
You're right, I didn't notice, when I copied the code snippet, that err is not defined